### PR TITLE
CXP-78 : [Front] Add loading spinner

### PR DESCRIPTION
**Pull request by tomglvng**
https://akeneo.atlassian.net/browse/CXP-78

- Replace "loading..." by a svg loading spinner.
image

- Fix bug linter `Container = styled(TableCell)` in `src/Akeneo/Connectivity/Connection/front/src/error-management/components/ErrorList/ErrorDateTimeCell.tsx`

### CODE REVIEW

File: src/Akeneo/Connectivity/Connection/front/src/common/icons/LoadingSpinnerIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import React, {SVGProps} from 'react';
+
+export const LoadingSpinnerIcon = (props: SVGProps<SVGSVGElement>) => (
```
Comment (pchasle) [inline src/Akeneo/Connectivity/Connection/front/src/common/icons/LoadingSpinnerIcon.tsx:3]: It's not an icon, I think you can put it inside `common/components/Loading/`
and have `LoadingSpinner.tsx` <= svg
`Loading.tsx` <= loading component
and only export the Loading component

**Comment by tomglvng** [comment]
After some CSS

image

File: src/Akeneo/Connectivity/Connection/front/src/common/icons/LoadingSpinnerIcon.tsx (hunk 1 above)
Comment (tomglvng) [reply src/Akeneo/Connectivity/Connection/front/src/common/icons/LoadingSpinnerIcon.tsx]: Done in https://github.com/akeneo/pim-community-dev/pull/12383/commits/0637782d0cf74e102bd5bab97da009374c129975

File: src/Akeneo/Connectivity/Connection/front/src/common/components/Loading/Loading.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const Container = styled.div`
+    height: 100%;
+    display: grid;
+`;
+
+const Spinner = styled(LoadingSpinner)`
+    background-size: 60px;
+    min-height: 60px;
+    background: center;
+    margin: auto;
```
Comment (pchasle) [inline src/Akeneo/Connectivity/Connection/front/src/common/components/Loading/Loading.tsx:22]:
```suggestion
    min-height: 60px;
    margin: auto;
```

File: src/Akeneo/Connectivity/Connection/front/src/common/components/Loading/Loading.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+        <Container>
+            <Spinner />
+        </Container>
+    );
+};
+
+const Container = styled.div`
+    height: 100%;
+    display: grid;
```
Comment (pchasle) [inline src/Akeneo/Connectivity/Connection/front/src/common/components/Loading/Loading.tsx:15]: It feels weird to have a css grid, any particular reason to have it here ?

Comment (tomglvng) [reply src/Akeneo/Connectivity/Connection/front/src/common/components/Loading/Loading.tsx]: You're right : I simplified it in https://github.com/akeneo/pim-community-dev/pull/12383/commits/f60e5088018faa35944d9615c3411e5f8a426e82

File: src/Akeneo/Connectivity/Connection/front/src/common/components/Loading/Loading.tsx (hunk 2 above)
Comment (tomglvng) [reply src/Akeneo/Connectivity/Connection/front/src/common/components/Loading/Loading.tsx]: Done in https://github.com/akeneo/pim-community-dev/pull/12383/commits/f60e5088018faa35944d9615c3411e5f8a426e82

**Pull request merged**